src/pages/public/about-us-page/main-value/MainValue.tsx

```diff
@@ -1,31 +1,38 @@
 import { ABOUT_US_DATA } from '../../../../const/public/about-us-page';
 import './MainValue.scss';
+import { CustomSwiper } from '../../../../components/public/swiper/CustomSwiper';
 
 export const MainValues = () => {
     return (
         <div className="main-values-block">
             <div className="main-values-title">
                 <h2>
-                    {ABOUT_US_DATA.MAIN_VALUE.FIRST_PART}
-                    <span>{ABOUT_US_DATA.MAIN_VALUE.FIRST_HIGHLIGHT}</span>
+                    {ABOUT_US_DATA.MAIN_VALUE.FIRST_PART} <br />
+                    <span>{ABOUT_US_DATA.MAIN_VALUE.FIRST_HIGHLIGHT}</span> <br />
                     {ABOUT_US_DATA.MAIN_VALUE.MIDDLE_PART} <br />
                     <span>{ABOUT_US_DATA.MAIN_VALUE.SECOND_HIGHLIGHT}</span>
```

Comment:
`<br>` is an accessibility anti-pattern. Screen readers may interpret these tags as unnatural pauses, and it's semantically incorrect to use HTML for styling.

Maybe we can use `display: block;` for this `span` instead of `<br>`? or come up with your ideas :)
```suggestion
                    {ABOUT_US_DATA.MAIN_VALUE.FIRST_PART}
                    <span>{ABOUT_US_DATA.MAIN_VALUE.FIRST_HIGHLIGHT}</span>
                    {ABOUT_US_DATA.MAIN_VALUE.MIDDLE_PART}
                    <span>{ABOUT_US_DATA.MAIN_VALUE.SECOND_HIGHLIGHT}</span>
```